### ISSUE TITLE
Bot's work

### ISSUE THREAD

**Issue by Batyr0212** [alexschimpf/Snkrs-Bot#56]
Will the bot work in several windows of browser in same time?

**Comment by jfleurimond** [comment]
Yes, I have done this. Only issue is NIKE limiting your orders if you have one pending order already for the same release. Probably will not be an issue if using multiple nike accounts
